packages/devtools-core/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@prodo/devtools-core",
+  "version": "0.0.6",
+  "main": "transpiled/index.js",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf .cache build dist lib tsconfig.tsbuildinfo",
+    "start": "webpack-dev-server --config webpack.config.js --mode=development --history-api-fallback --hot",
```

Comment:
What does starting the devtools by itself do?

Reply:
In initial stages of development I used it to render the devtools on its own, but I don't use it anymore - will remove the command.